core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1003,14 +1006,15 @@ private long completeFileAndJournal(LockedInodePath inodePath, CompleteFileOptio
     // determined by its memory footprint.
     long length = fileInode.isPersisted() ? options.getUfsLength() : inMemoryLength;
 
-    completeFileInternal(fileInode.getBlockIds(), inodePath, length, opTimeMs);
+    completeFileInternal(fileInode.getBlockIds(), inodePath, length, options.getOperationTimeMs());
     CompleteFileEntry completeFileEntry = CompleteFileEntry.newBuilder()
         .addAllBlockIds(fileInode.getBlockIds())
         .setId(inode.getId())
         .setLength(length)
-        .setOpTimeMs(opTimeMs)
+        .setOpTimeMs(options.getOperationTimeMs())
         .build();
-    return appendJournalEntry(JournalEntry.newBuilder().setCompleteFile(completeFileEntry).build());
+    flushCounter.setValue(
+        appendJournalEntry(JournalEntry.newBuilder().setCompleteFile(completeFileEntry).build()));
```

Comment:
what does "more type-safe" mean? similar comment as for waitForJournalFlush (i.e. it is used in BlockMaster which does not adopt this idiom so I kept the signature the same)